images/metalk8s-ui/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG NGINX_IMAGE_VERSION=1.15.8
+
+FROM nginx:${NGINX_IMAGE_VERSION}
+
+COPY metalk8s-ui-nginx.conf /etc/nginx/conf.d/default.conf
+
+RUN rm -rf /usr/share/nginx/html/*
+
+COPY ui /usr/share/nginx/html/
+COPY docs/html /usr/share/nginx/docs
```

Comment:
No, not fond of this. `docs` are part of *DocumentRoot* as is `ui`. So, copy `ui` into `/usr/share/nginx/html`, copy `docs` into `/usr/share/nginx/html`, set up `/usr/share/nginx/html` as the server root, and that should basically be it.

Reply:
Ok, wasn't sure of how we want to do this, will do.